src/main/webapp/team.html

```diff
@@ -156,61 +180,53 @@ <h5 class="title">
-                    });
-                    $("#btnShowMore2").click(function () {
-                        let contentPartTwo = Mustache.render($("#templateTeam").html(), {"data": partTwo});
-                        $("#teamContent").append(contentPartTwo);
-                        document.getElementById("btnShowMore").style.display = "none";
-                    });
-                    $("#btnShowMore3").click(function () {
-                        let contentPartTwo = Mustache.render($("#templateTeam").html(), {"data": partTwo});
-                        $("#teamContent").append(contentPartTwo);
-                        document.getElementById("btnShowMore").style.display = "none";
-                    });
-                } else {
-                    //mustache render
-                    let content = Mustache.render($("#templateTeam").html(), {"data": data});
-
-                    //display first 8 profiles
-                    $("#teamContent").html(content);
-                    $("#teamContent1").html(content);
-                    $("#teamContent2").html(content);
-                    $("#teamContent3").html(content);
-
-                    //hide button
-                    $("#btnShowMore").style.display = "none";
-                    $("#btnShowMore1").style.display = "none";
-                    $("#btnShowMore2").style.display = "none";
-                    $("#btnShowMore3").style.display = "none";
+            success: function (payload) {
+                let advisoryPanel = payload.advisoryPanel;
+                let consultants = payload.consultants;
+                let externalCollaborators = payload.externalCollaborators;
+                let executiveCommitee = payload.executiveCommitee;
+                let techTeam = payload.techTeam;
+                let designTeam = payload.designTeam;
+
+                loadData(advisoryPanel, "advisoryPanel");
+                loadData(consultants, "consultants");
+                loadData(externalCollaborators, "externalCollaborators");
+                loadData(executiveCommitee, "executiveCommitee");
+                loadData(techTeam, "techTeam");
+                loadData(designTeam, "designTeam");
+
+                function loadData(data, contentId) {
+                    //slice array to two parts
+                    if (data.length >= 8) {
+                        let partOne = data.slice(0, 8);
+                        let partTwo = data.slice(8);
+
+                        //mustache render - part one
+                        let contentPartOne = Mustache.render($("#templateTeam").html(), {"data": partOne});
+
+                        //display first 8 profiles
+                        $("#"+contentId).html(contentPartOne);
+
+                        //hide button
+                        $("#btnShowMore_"+contentId).click(function () {
+                            let contentPartTwo = Mustache.render($("#templateTeam").html(), {"data": partTwo});
```

Comment:
Instead of introducing variables (`partOne`, `partTwo`) let's use 
```
let contentPartTwo = Mustache.render($("#templateTeam").html(), {"data": profiles.slice(8)});
```
It is easier to understand I guess.